game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProTerritoryManager.java

```diff
@@ -119,12 +119,9 @@ public void populateEnemyDefenseOptions() {
     for (final Territory t : attackMap.keySet()) {
       final ProTerritory patd = attackMap.get(t);
 
-      // Check if I can win without amphib units and ignore AA since max units might have lots of planes
-      List<Unit> defenders = CollectionUtils.getMatches(patd.getMaxEnemyDefenders(player, data),
-          ProMatches.unitIsEnemyAndNotAa(player, data));
-      if (isIgnoringRelationships) {
-        defenders = new ArrayList<>(t.getUnits());
-      }
+      // Check if I can win without amphib units
+      final List<Unit> defenders =
+          new ArrayList<>(isIgnoringRelationships ? t.getUnits() : patd.getMaxEnemyDefenders(player, data));
```

Comment:
So removing all units with AA attacks causes issues for advanced maps that have lots of different types of unit with AA style attacks. So remove this filter as its causing problems.